src/commands/playbook-publish.test.ts

```diff
@@ -0,0 +1,26 @@
+import { describe, it } from '@jest/globals'
+import { getProgramMock, getTempFolder } from '../utils/test-utils'
+import publishPlaybook from './playbook-publish'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import assert from 'node:assert'
+
+const programMock = getProgramMock()
+
+describe('playbook publish', () => {
+    it('FAIL - playbook not exist', async () => {
```

Comment:
missing cases:

* FAIL - playbook yml is invalid
* FAIL - playbook.yml doesnt exist
* PASS - playbook created.